main/main.cpp

```diff
@@ -1082,6 +1083,8 @@ bool Main::start() {
 			bool parsed_pair=true;
 			if (args[i]=="-doctool") {
 				doc_tool=args[i+1];
+				for(int j=i+2; j<args.size();j++)
+					removal_docs.push_back(args[j]);
```

Comment:
Here is a checking on line 1446, so it should be ok :)
`if (rmdoc.load(E->get()) == OK)`